src/components/sbb-journey-summary/sbb-journey-summary.stories.jsx

```diff
@@ -47,14 +47,28 @@ const Template = ({ 'disable-animation': disableAnimation, 'data-now': dataNow,
 );
 
 const TemplateNoSlot = ({
+  'disable-animation': disableAnimation,
+  'data-now': dataNow,
+  trip: trip,
+}) => (
+  <sbb-journey-summary
+    disable-animation={disableAnimation}
+    data-now={dataNow}
+    trip={trip}
+  ></sbb-journey-summary>
+);
+
+const TemplateRoundtrip = ({
   'disable-animation': disableAnimation,
   'data-now': dataNow,
   ...args
 }) => (
   <sbb-journey-summary
     disable-animation={disableAnimation}
     data-now={dataNow}
-    config={args}
+    trip={args.trip}
+    tripBack={args.tripBack}
+    round-trip={args.roundTrip}
   ></sbb-journey-summary>
 );
```

Comment:
"TemplateRoundtrip" can probably removed and replaced by the simplified "TemplateNoSlot"